Dockerfile

```diff
@@ -0,0 +1,87 @@
+# Set environment
+ARG BASE_DIR=/opt/cmis-server
+
+#####
+# Preparation stage 
+#####
+FROM exoplatform/jdk:8-ubuntu-1804 AS install
+
+ARG BASE_DIR
```

Comment:
its the way I used to avoid to redeclare on each ``FROM`` the ``BASE_DIR``. As there is no value provided, the value line 2 is used. Without this, ``BASE_DIR`` is not known on the FROMs